src/material.rs

```diff
@@ -157,6 +157,17 @@ impl<'a> Material<'a> {
         self.json.emissive_factor.0
     }
 
+    /// Specifies whether the material is unlit.
+    /// 
+    /// * When this value is false, primitive should be renderer using Physically-Based Rendering (PBR) methodology.
+    /// * When this value is true, primitive should not be renderer using Physically-Based Rendering (PBR) methodology.
```

Comment:
The `KHR_materials_unlit` is a suggestion to not use the 'core' PBR values. https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_unlit#overview

I suggest we reword something like:

```
Returns `true` if the [`KHR_materials_unlit`] property was specified, in which case the renderer should prefer to ignore all PBR values except `baseColor`.

[`KHR_materials_unlit`](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_unlit#overview)
```

Reply:
You're right, it's just a suggestion, I changed it. Thanks.